CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Changelog
 
+## 1.9.0
+* Feature - Add Amazon CloudWatch Logs logging driver.
+* Bug - Fixed ACS handler when acking blank message ids.
+* Bug - Fixed incorrect reporting of CPU utilization.
```

Comment:
Can you rephrase these?

* Bug - Fixed an issue where CPU utilization could be reported incorrectly.
* Bug - Resolved a bug where containers would not get cleaned up in some cases.